src/main/flight/pid.h

```diff
@@ -76,6 +76,12 @@ typedef struct pid8_s {
     uint8_t D;
 } pid8_t;
 
+typedef enum {
+    ANTI_GRAVITY_OFF = 0,
+    ANTI_GRAVITY_STEP,
+    ANTI_GRAVITY_SMOOTH
```

Comment:
Imho better name for SMOOTH mode should be FILTER mode

Reply:
I don't mind the *internal* name being ANTI_GRAVITY_FILTER. However, the outcome is a continuously variable increase in I in proportion to an increasing rate of change of throttle.  This provides a smoother range of effect that doesn't happen with the all-or-none threshold based method.  I'd prefer the user name in the CLI to reflect what the user experience will be, not the technical method used to achieve that result.

Reply:
Ok no problem i think entire STEP mode will be already removed soon to save flash space because yor SMOOTH mode working well and its the way to go ;)